radiation/python/config.py

```diff
@@ -1,11 +1,9 @@
 import numpy as np
 import sys
 
-sys.path.insert(
-    0, "/Users/AndrewP/Documents/work/physics_standalone/radiation/python/radlw"
-)
-from radlw_param import nbands, maxgas, maxxsec, ngptlw
-from radsw_param import ngptsw, nbhgh, nblow, nbdsw
+sys.path.insert(0, "/work/radiation/python")
+from radlw.radlw_param import nbands, maxgas, maxxsec, ngptlw
+from radsw.radsw_param import ngptsw, nbhgh, nblow, nbdsw
```

Comment:
this convention works great and you don't need to change it here. It can sometimes be helpful for a code reader to instead do something like `import radlw.radlw_param as radlw_param`, and then refer to the variables as e.g. `radlw_param.nbands`, so that while reading the code it is very obvious what nbands is. But, that can also sometimes make lines too verbose an long, always a tradeoff!